### DISCUSSION TITLE
MariaDB support

### DISCUSSION THREAD

**Discussion by robertsilen** [bacula-web/bacula-web#225] [Q&A] [answered]
Hello, any reason MariaDB is not mentioned as a support database? I see you are linking to the MySQL PHP documentation page at https://www.php.net/manual/en/set.mysqlinfo.php, which doesn't mention MariaDB. I asked about that now in  https://github.com/php/doc-en/issues/3628 , but wondering if bacula-web has any reason not to support MariaDB? Thanks for any comments on this. 

This is part of my efforts I describe in my blog [Improving MariaDB Support in Open Source Projects](https://mariadb.org/improving-mariadb-support-in-open-source-projects/).

**Comment by dfranco** [comment]
Hello @robertsilen 

Thanks to notice it, this is just an oversight on my part.

Bacula open source project's documentation is a bit outdated, and it's really hard to find something in it.
As far as I can remember, both MySQL and MariaDB are supported, so does Bacula-Web ...

Best,

**Comment by dfranco** [comment]
Hi @robertsilen,

As it looks to be more a question/feedback than a proper issue or feature request, I've converted this one to a discussion.

A specific issue will be created to update current documentation.

Best,

**Comment by robertsilen** [comment]
Cool, MariaDB was now added to the [PHP MySQL Drivers and Plugins](https://www.php.net/manual/en/set.mysqlinfo.php) page that the README links to. So made a PR to add MariaDB to the readme at [#226](https://github.com/bacula-web/bacula-web/pull/226). Hope thats ok? :)

**Reply by dfranco** [reply]
Awesome, I'll have a look later at the PR

**Comment by dfranco** [comment, marked as answer]
Hi @robertsilen ,

the PR #226 is now merged.

Thanks a lot for your contribution

**Comment by dfranco** [comment]
Updated documentation related to above PR in [5110200](https://github.com/bacula-web/www.bacula-web.org/commit/51102006e48682fa8e205fb69ab1b3bf2207ca0e)
